ORBIT/manager.py

```diff
@@ -1438,38 +1509,206 @@ def overnight_capex(self):
+            num_turbines = self.config["plant"]["num_turbines"]
+            rating = self.config["turbine"]["turbine_rating"]
+            project_completion = project_completion_per_kW * num_turbines * rating * 1000
+            
+        except:
+            project_completion_factor =  self.project_params.get("project_completion_factor", 0.0115)
+            project_completion = (self.turbine_capex + self.bos_capex + self.project_capex) *\
+                                    project_completion_factor
+
+        return project_completion
+
+    @property
+    def procurement_contingency_capex(self):
+        """
+        Returns the procurement contingency capital cost of the project.
+        Methodology from ORCA model, default values used in 2022 Cost of Wind Energy Review.
+        """
+
+        try:
+            procurement_contingency_per_kW = self.config["project_parameters"]["procurement_contingency"]
+            num_turbines = self.config["plant"]["num_turbines"]
+            rating = self.config["turbine"]["turbine_rating"]
+            procurement_contingency = procurement_contingency_per_kW * num_turbines * rating * 1000
+            
+        except:
+            procurement_contingency_factor =  self.project_params.get("procurement_contingency_factor", 0.0575)
+            procurement_contingency = (self.turbine_capex + self.bos_capex + self.project_capex - self.installation_capex) *\
+                                    procurement_contingency_factor
+
+        return procurement_contingency
+
+    @property
+    def installation_contingency_capex(self):
+        """
+        Returns the installation contingency capital cost of the project.
+        Methodology from ORCA model, default values used in 2022 Cost of Wind Energy Review.
+        """
+
+        try:
+            installation_contingency_per_kW = self.config["project_parameters"]["installation_contingency"]
+            num_turbines = self.config["plant"]["num_turbines"]
+            rating = self.config["turbine"]["turbine_rating"]
+            installation_contingency = installation_contingency_per_kW * num_turbines * rating * 1000
+            
+        except:
+            installation_contingency_factor =  self.project_params.get("installation_contingency_factor", 0.345)
+            installation_contingency = self.installation_capex * installation_contingency_factor
+
+        return installation_contingency
+
+    @property
+    def construction_financing_factor(self):
+        """
+        Returns the construction finaning factor of the project.
+        Methodology from ORCA model, default values used in 2022 Cost of Wind Energy Review,
+        except the spend schedule, which is sourced from collaborations with industry.
+        """
+
+        try:
+            spend_schedule = self.config["project_parameters"]["spend_schedule"]
+        except:
+            spend_schedule =  self.project_params.get("spend_schedule", {0: 0.25,
```

Comment:
This try/except is redundant. The line 1653 is essentially "trying to get" `spend_schedule` from the project_params dictionary. If its not there, then you specify the values of the spend schedule. I think you can remove lines 1650, 51, 52.

Reply:
It works without the try/except. Thanks for the suggestion.